src/GenerateAnalyzerRulesets/Program.cs

```diff
@@ -268,13 +268,22 @@ void createPropsFile()
 
                 var fileContents =
 $@"<Project DefaultTargets=""Build"" xmlns=""http://schemas.microsoft.com/developer/msbuild/2003"">
-  {getEditorConfigAsAdditionalFile()}{getCodeAnalysisTreatWarningsNotAsErrors()}{getRulesetOverrides()}
+  {getEditorConfigAsAdditionalFile()}{getCodeAnalysisTreatWarningsNotAsErrors()}{getRulesetOverrides()}{getFlowAnalysisFeatureFlag()}
 </Project>";
                 var directory = Directory.CreateDirectory(propsFileDir);
                 var fileWithPath = Path.Combine(directory.FullName, propsFileName);
                 File.WriteAllText(fileWithPath, fileContents);
             }
 
+            string getFlowAnalysisFeatureFlag()
+            {
+                return @"
+
+  <PropertyGroup>
+    <Features>$(Features);flow-analysis</Features> 
```

Comment:
```suggestion
    <Features>$(Features), flow-analysis</Features> 
```

Reply:
Hmm, this is how we use it in this repo
https://github.com/dotnet/roslyn-analyzers/blob/5ed057905c4e4def7f9e5abd4baccf8fc7286b4a/src/Directory.Build.props#L10

Reply:
https://github.com/dotnet/roslyn/blob/4ee95ab9ea1e5b6625912e284af342cffe7cf24e/eng/targets/Settings.props#L23

Reply:
OK, it seems either way is fine

http://source.roslyn.io/#Microsoft.Build.Tasks.CodeAnalysis/CompilerOptionParseUtilities.cs,21

Reply:
Also, why is IOperation feature flag is still used in Roslyn? Should remove that

Reply:
I will leave it as-is here. Do you have any other comments?